configure.ac

```diff
@@ -1036,6 +1054,10 @@ AM_CONDITIONAL([ENABLE_BENCH],[test x$use_bench = xyes])
 AM_CONDITIONAL([USE_QRCODE], [test x$use_qr = xyes])
 AM_CONDITIONAL([USE_LCOV],[test x$use_lcov = xyes])
 AM_CONDITIONAL([ENABLE_GPERF],[test x$use_gperf = xyes])
+AM_CONDITIONAL([ENABLE_GPROF],[test x$use_gprof = xyes])
+AM_CONDITIONAL([ENABLE_MUTRACE],[test x$use_mutrace = xyes])
+AM_CONDITIONAL([USE_COMPARISON_TOOL],[test x$use_comparison_tool != xno])
+AM_CONDITIONAL([USE_COMPARISON_TOOL_REORG_TESTS],[test x$use_comparison_tool_reorg_test != xno])
```

Comment:
we removed the above two lines in ff4ad40c9dcbad6fdb66b6ac67043aaef581de64

Reply:
thx! good catch